index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <style>
+      @media only screen and (max-width: 600px) {
+        body {
+          background-color: lightblue;
+        }
+      }
+    </style>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="welcome-section">
+      <h1>This is a portfolio</h1>
+    </section>
+    <section id="projects">
+      <div class="project-tile">project title</div>
+      <a href="https://www.weatherapi.com/">My weather API</a>
+    </section>
+    <navbar id="navbar">
+      <a href="#projects">Projects</a>
+      <br />
+    </navbar>
+    <a
+      href="https://github.com/codedit334/microverse_trials_emmanuel_oussama"
+      id="profile-link"
+      target="_blank"
+      >GitHub</a
+    >
+
+    <script src="js/index.js"></script>
+  </body>
+</html>
```

Comment:
- [ ] It better to remove spaces in your code, It makes code look professional.

Reply:
- [ ] In terms of clarity, the comment/suggestion you provided is clear and to the point. It provides a recommendation to remove spaces in the code, which can improve the professionalism of the code. However, the **wording and grammar** could be improved. Here is a suggested rephrased comment with corrected grammar:

  "It **is** better to remove spaces in your code to improve its professional appearance." 🙋‍♂️

  _**Use this as an example and tailor it to your own feedback.**_ 🙌

Reply:
Done